lib/engine/game/g_18_gb/entities.rb

```diff
@@ -298,7 +293,6 @@ module Entities
               },
               {
                 type: 'hex_bonus',
-                when: 'owning_playing_or_turn',
                 owner_type: 'player',
```

Comment:
I talked to 18gb implementer, there's a typo ('owning_playing' vs 'owning_player'), when isn't needed for the abilities in this game